scripts/setup.mongodb

```diff
@@ -10,14 +32,64 @@ try {
 ["ims", "test-ims"].forEach((databaseName) => {
   db = db.getSiblingDB(databaseName);
 
+  // Add some usage statuses if the collection doesnt already exist
+  if (!db.getCollectionNames().includes("usage_statuses")) {
+    console.log(`Populating usage statuses in ${databaseName}...`);
+
+    db.usage_statuses.insertMany([
+      createUsageStatusIn(USAGE_STATUS_ID_NEW, "New"),
+      createUsageStatusIn(USAGE_STATUS_ID_IN_USE, "In Use"),
+      createUsageStatusIn(USAGE_STATUS_ID_USED, "Used"),
+      createUsageStatusIn(USAGE_STATUS_ID_SCRAPPED, "Scrapped"),
+    ]);
+  }
+
   // Add some system types if the collection doesn't already exist
   if (!db.getCollectionNames().includes("system_types")) {
     console.log(`Populating system types in ${databaseName}...`);
 
     db.system_types.insertMany([
-      { _id: ObjectId("685e5dce6e347e39d459c5ea"), value: "Storage" },
-      { _id: ObjectId("685e5dce6e347e39d459c5eb"), value: "Operational" },
-      { _id: ObjectId("685e5dce6e347e39d459c5ec"), value: "Scrapped" },
+      { _id: SYSTEM_TYPE_ID_STORAGE, value: "Storage" },
+      { _id: SYSTEM_TYPE_ID_OPERATIONAL, value: "Operational" },
+      { _id: SYSTEM_TYPE_ID_SCRAPPED, value: "Scrapped" },
+    ]);
+  }
+
+  // Add some rules if the collection doesn't already exist
+  if (!db.getCollectionNames().includes("rules")) {
+    console.log(`Populating rules in ${databaseName}...`);
+
+    db.rules.insertMany([
+      {
+        // Creation of items in storage -> New
+        src_system_type_id: null,
+        dst_system_type_id: SYSTEM_TYPE_ID_STORAGE,
+        dst_usage_status_id: USAGE_STATUS_ID_NEW,
```

Comment:
Is `dst_` needed before `usage_status_id`? There is no `src_usage_status_id` so I am not sure if it is needed.

Reply:
We discussed this and agreed to keep as it is.

Reply:
Just repeating what I said for future reference: "Originally I just left it out, then added it thinking without it just says usage status without explaining what it means, and I felt putting dst_ there kind of suggests what its for - it defines the usage status at the destination. There isnt a src one since its irrelevant and will just be overridden."